api/src/main/java/org/eclipse/microprofile/config/spi/ConfigSource.java

```diff
@@ -31,7 +31,7 @@
  * <p>A ConfigSource will get picked up via the
  * {@link java.util.ServiceLoader} mechanism and must get registered via
  * META-INF/services/javax.config.spi.ConfigSource</p>
- * The other custom config source can be added programmatically via {@link org.eclipse.microprofile.config.ConfigProvider}.
+ * The other custom config source can be added programmatically via {@link org.eclipse.microprofile.config.ConfigProvider}. << How?
```

Comment:
I understand. I think that the javadoc should be improved to avoid confusion because it mixes information about 2 different ways to access them, which cannot be mixed. It should clearly say that a ConfigSource is registered either by the default config using service loader, or registered programmatically to a new config created by a builder. I would also move the details about service loader to ConfigProvider, because it is not directly related to the interface itself, rather to the ConfigProvider that loads it.